src/foam/nanos/auth/ChangePasswordView.js

```diff
@@ -49,13 +50,14 @@ foam.CLASS({
     }
 
     ^ .top-bar {
+      background: /*%PRIMARY1%*/
```

Comment:
If FOAM is run in a non-nanos app, then views won't have theme and this won't be replaced properly, so it's best practice to write it like this if you want to use theme properties in CSS, which is recommended:

```suggestion
      background: /*%PRIMARY1%*/ #202341;
```

That way in non-nanos FOAM apps, the CSS falls back on the default value.